roles/nextflow/defaults/main.yml

```diff
@@ -3,11 +3,11 @@ nextflow_java: "{{ java_home }}"
 nextflow_version_tag: 21.04.3
 nextflow_download_url: https://github.com/nextflow-io/nextflow/releases/download/v{{ nextflow_version_tag }}/nextflow
 nextflow_env:
-  NXF_LAUNCHER: /scratch/{{ deployment_version }}
-  NXF_LAUNCHBASE: /scratch/{{ deployment_version }}
+  NXF_LAUNCHER: $HOME/.nextflow/work
+  NXF_LAUNCHBASE: $HOME/.nextflow/work
   NXF_HOME: "{{ nextflow_dest }}/workfiles"
-  NXF_TEMP: /scratch/{{ deployment_version }}/tmp
-  NXF_WORK: /scratch/{{ deployment_version }}/work
+  NXF_TEMP: $HOME/.nextflow/work
+  NXF_WORK: $HOME/.nextflow/work
```

Comment:
This is the users `$HOME` right?

Reply:
@ewels Pär kept getting errors like `nextflow: line 457: /scratch/classpath-f13425eb805ad83a07ff0147b535d5fc: Permission denied` and I was assuming there was problems related to `/scratch` being local to the processing node but maybe that's not the case.. good input though, I'll see what I can do.

Reply:
Yeah, I wondered if there might be cross-node issues with using scratch too (it's common practice to use scratch within a task but work directories should be accessible from everywhere that a nextflow process can run).

I think a dedicated shared work directory somewhere on the networked storage is best. It might also be smart to set `cleanup = true` in our Nextflow config, which automatically deletes the work directories after a successful run. https://nextflow.io/docs/latest/config.html#miscellaneous - no ability to resume but I'm not sure how much we use that? (for successful runs anyway)